db/osqlcomm.c

```diff
@@ -7547,8 +7547,8 @@ int osql_process_packet(struct ireq *iq, uuid_t uuid, void *trans, char **pmsg,
 
         if (rc != 0) {
             if (rc != RC_INTERNAL_RETRY) {
-                errstat_cat_strf(&iq->errstat,
-                                 " unable to update record rc = %d", rc);
+                errstat_cat_strf(&iq->errstat," unable to update record");
+                logmsg(LOGMSG_ERROR, "Failed to update record. Internal error code is %d\n", rc);
```

Comment:
We might need this for diagnosing an issue if we send 'generic internal error' to the client